lib/rules/no-compare-neg-zero.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @fileoverview The rule should warn against code that tries to compare against -0.
+ * @author no-compare-neg-zero
```

Comment:
This line should have your name (preferably your full name, but I think your GitHub username could also be acceptable).